File: JAVASCRIPT PROGRAMMING/Test63.js
"use strict";
// 63. STRONG NUMBER
const StrongNumber = (num) => {
    let temp = 1, NUM = num, count = 0;
    while (num !== 0) {
        let rem = num % 10;
        for (let i = 1; i <= rem; i++) {
            temp *= i;
        }
        count += temp;
        temp = 1;
        num = Math.floor(num / 10);
    }
    if (NUM === count) {
        console.log(`${NUM} Is A Strong Number.`);
    }
    else {
        console.log(`${NUM} Is Not A Strong Number.`);
    }
};
StrongNumber(145);
StrongNumber(265);

File: JAVA PROGRAMMING/Test63.java
// 63. STRONG NUMBER
import java.util.Scanner;
public class Test {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);

        System.out.println("Enter Num Value: ");
        int num = in.nextInt();

        int temp = 1, NUM = num, count = 0;
        while(num != 0) {
            int rem = num % 10;
			for(int i = 1; i <= rem; i++)	{
				temp *= i;
			}
			count += temp;
			temp = 1;
			num /= 10;
        }
        if(NUM == count) {
            System.out.println(NUM+" Is Strong Number.");
        } else {
            System.out.println(NUM+" Is Not Strong Number.");
        }
        in.close();
    }
}